crates/sui-core/src/authority.rs

```diff
@@ -1254,32 +1253,37 @@ impl AuthorityState {
             epoch_store.protocol_config(),
         );
         let (kind, signer, _) = transaction.execution_parts();
+
+        let silent = true;
         // don't bother with paranoid checks in dry run
         let enable_move_vm_paranoid_checks = false;
```

Comment:
Noticed a discrepency here, between dev inspect and dry run -- the former runs with paranoid checks enabled if the node is configured that way, whereas the latter does not.  Is that intentional? cc @tnowacki

Reply:
I am surprised by this and I think we should remove it.
If that was for dev inspect I could relate more but for dry run I would expect all checks to be identical to those of a regular run.
I am curious to know if there is a reason for it

Reply:
Agreed -- I will do it in a follow-up.